router/src/server.rs

```diff
@@ -733,6 +782,17 @@ async fn metrics(prom_handle: Extension<PrometheusHandle>) -> String {
     prom_handle.render()
 }
 
+async fn request_logger(mut rx: mpsc::Receiver<(i64, String, String)>) {
+    // log the function is getting called
+    let mut file = tokio::fs::File::create("/data/log.txt").await.unwrap();
```

Comment:
Is there a way to ensure that this log file doesn't grow too large? E.g. maybe we partition the logs based on time, and can have a way to delete the old files?